src/Modules/ModParser.lua

```diff
@@ -4281,7 +4281,7 @@ local specialModList = {
 		mod("EnemyModifier", "LIST", { mod = mod("ColdResist", "INC", num) }),
 	} end,
 	["nearby enemies are blinded while physical aegis is not depleted"] = { mod("EnemyModifier", "LIST", { mod = flag("Condition:Blinded") }, { type = "Condition", var = "PhysicalAegisDepleted", neg = true }) },
-	["energy shield is increased by chance to block spell damage"] = { flag("EnergyShieldIncreasedByChanceToBlockSpellDamage") },
```

Comment:
You actually don't need to capitalize any parsing in ModParser, as all mods are lowercased beforehand.  In addition, the wording without "maximum" doesn't exist anywhere, so we don't even need this optional bit.  I'll just remove it myself before merging